### PR TITLE
updating testnet-2 concluded post

### PR DESCRIPTION
**Pull request by b00f**
This PR updates the Testnets concluded posts

### CODE REVIEW

File: website/_posts/2024-01-22-testnet-2-concluded.md [hunk 1]
```diff
@@ -15,4 +15,60 @@ If you like to read more about the Testnet you can check out our post on
...
+the development team shifted its focus to the Mainnet launch.
+
+## Special thanks to the community
+
+This Testnet was important for us.
+It helped us learn what we need to do to make Pactus work even better.
+Without the community that supports us and dedicates their time and resources to testing and feedback,
+we wouldn't have been able to progress as we have.
+
+we thank all of them for their valuable contributions.
```
Comment (Mr-HoDL58) [inline website/_posts/2024-01-22-testnet-2-concluded.md:69]: We**

File: website/_posts/2023-04-21-testnet-0-concluded.md [hunk 2]
```diff
@@ -36,8 +36,17 @@ At this point, the development team decided to **end the Testnet-0** and move on
...
-We are grateful to all those who participated in Testnet-0, and we encourage them to join our next Testnet to
-help us improve Pactus.
+## Special thanks to the community
+
+This Testnet was important for us.
+It helped us learn what we need to do to make Pactus work even better.
+Without the community that supports us and dedicates their time and resources to testing and feedback,
+we wouldn't have been able to progress as we have.
+
+we thank all of them for their valuable contributions.
```
Comment (Mr-HoDL58) [inline website/_posts/2023-04-21-testnet-0-concluded.md:46]: We**

**Review by Mr-HoDL58 — COMMENTED** [review]
Just a few typos to correct then all good.

File: website/_posts/2024-01-22-testnet-2-concluded.md (hunk 1 above)
Comment (Mr-HoDL58) [reply website/_posts/2024-01-22-testnet-2-concluded.md:69]: line 69

File: website/_posts/2023-04-21-testnet-0-concluded.md (hunk 2 above)
Comment (Mr-HoDL58) [reply website/_posts/2023-04-21-testnet-0-concluded.md:46]: line 46

**Comment by Mr-HoDL58** [comment]
It has the word "coind" in the image.
image

**Comment by b00f** [comment]
@Mr-HoDL58 You have the eyes of a hawk.
Thanks for finding this mistake.

**Pull request merged**